src/pages/index.tsx

```diff
@@ -1,11 +1,30 @@
+import { useForm } from 'react-hook-form'
 import { Button, Input } from '@/components/common'
 import { TextArea } from '@/components/common/TextArea'
 import { MainLayout } from '@/components/layout/MainLayout'
+import { Calender } from '@/components/screen'
+import { Schedule } from '@/type/common'
 
 export default function Home() {
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+    setValue,
+  } = useForm<Schedule>({
+    mode: 'onSubmit',
+  })
+
+  const onSubmit = (data: Schedule) => {
+    console.log(data)
+  }
+
   return (
     <MainLayout>
-      <div className='my-10 flex flex-col items-center gap-16 md:my-16'>
+      <form
+        className='my-10 flex flex-col items-center gap-16 md:my-16'
+        onSubmit={handleSubmit(onSubmit)}
+      >
         <div className='m-auto flex w-4/5 flex-wrap md:flex-nowrap md:gap-16'>
           <div className='flex flex-col md:w-1/2'>
```

Comment:
spだとイベント名とメモ欄が縮んで、dividerがカレンダーより小さくなってアンバランスになっているので、sp版だけw-fullの方が綺麗だと思います
```suggestion
          <div className='flex w-full flex-col md:w-1/2'>
```

Reply:
[13c2c75](https://github.com/NUTFes/chosei-chan/pull/46/commits/13c2c7543f06613839372a29fb8bb881acb0272c)